experimental/JuLie/quantum_analogs.jl

```diff
@@ -0,0 +1,295 @@
+################################################################################
+# Quantum analogs
+#
+# Copyright (C) 2020 Ulrich Thiel, ulthiel.com/math
+#
+# Mar 2023: Imported into OSCAR by UT
+#
+# Future features could include:
+# - Symmetric quantum numbers
+# - Two-colored quantum numbers
+################################################################################
+
+export quantum_integer, quantum_factorial, quantum_binomial
+
+################################################################################
+# Quantum integers
+################################################################################
+@doc Markdown.doc"""
+    quantum_integer(n::IntegerUnion, q::RingElem)
+    quantum_integer(n::IntegerUnion, q::Integer)
+    quantum_integer(n::IntegerUnion)
+    
+Let ``n ∈ ℤ`` and let ``ℚ(𝐪)`` be the fraction field of the polynomial ring ``ℤ[𝐪]`` in
+one variable ``𝐪``. The **quantum integer** ``[n]_𝐪 ∈ ℚ(𝐪)`` of ``n`` is defined as
```

Comment:
But `quantum_integer(5,1)` is not in ℚ(𝐪) ... Also this is quite technical. Can't we just write this:
```suggestion
For an integer `n` and a ring element `q`, the **quantum integer** ``[n]_q`` is defined as
```

Reply:
In general I would suggest to minimize the use of Unicode. Most of it seems not necessary here. E.g. why does `q` have to be in (faux) bold face?